src/index.ts

```diff
@@ -13,8 +13,13 @@ import createImpl, {
 export * from './vanilla'
 
 // For server-side rendering: https://github.com/react-spring/zustand/pull/34
-const useIsoLayoutEffect =
-  typeof window === 'undefined' ? useEffect : useLayoutEffect
+// Deno support: https://github.com/pmndrs/zustand/issues/347
+const isSSR =
+  typeof window === 'undefined' ||
+  typeof window.navigator === 'undefined' ||
```

Comment:
```suggestion
  window.navigator === undefined ||
```
...is the same as: 
```js
  typeof window.navigator === 'undefined' ||
```

Reply:
nice. this should shave some bytes. how about, `!window.navigator`?

Reply:
Yes, even better

Reply:
thanks. let me fix the three prs.